subfunctions/g_CaBBI_observation

```diff
@@ -0,0 +1,19 @@
+-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+-% observation function:
+-%               
+-%               function [F] = g_CaBBI_observation(Xt,Phi,I_inp,inG)
+-%
+-% this function maps [Ca2+] kinetics to fluorescence observations (F) 
+-% through a non-linear saturating function
+-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+-
+-function [F] = g_CaBBI_observation(Xt,Phi,I_inp,inG)
```

Comment:
It would be better to have this function simply called `g_CaBBI`, as the fact it is an observation function is already implied by the `g_`prefix. To be changed in the demo scripts too.